core/editor/src/input/input_mapper.rs

```diff
@@ -103,6 +103,7 @@ macro_rules! mapping {
 impl Default for Mapping {
 	fn default() -> Self {
 		let (up, down, pointer_move) = mapping![
+			entry! {action=DocumentMessage::PasteLayers, key_down=KeyV, modifiers=[KeyControl]},
```

Comment:
Shouldn't this one be down near the bottom with Duplicate and Copy?

Reply:
I had to add the control-V shortcut to the beginning of the `mapping!`  because otherwise the `SelectTool(ToolType::Select)` message bound to V did take precedence.

Reply:
Ah, makes sense. Hopefully we can implement the precedence system sooner than later to deal with that more cleanly, but currently art features are a higher priority. Thanks for the explanation!